jwst/associations/lib/rules_level3_base.py

```diff
@@ -478,6 +478,21 @@ def ok_candidate(self, member=None):
         """
         return self.acid.type.lower() not in INVALID_AC_TYPES
 
+    def finalize(self):
+        if self.acid.type.lower() == 'group':
```

Comment:
This is possibly too restrictive? In fact, I wonder if it should be the other way: `acid.type != 'observation'`. All other candidate types should be multi-observation, though coronagraphic/ami should be reviewed. There is a possibility that those modes could be still valid for single observations.